test/libsolidity/ASTJSON/function_type_legacy.json

```diff
@@ -100,7 +100,7 @@
 															{
 																"constant" : false,
 																"name" : "",
-																"scope" : 16,
```

Comment:
Thanks for adding this to isoltest, @ekpyron ;)

Reply:
Although it's still a bit hacky, e.g. it's adding the ``// ----`` delimiter above, although it's not needed for ASTJSON tests - but I'll fix all that in 0.5.1.